model/span.go

```diff
@@ -25,17 +25,23 @@ import (
 	"go.uber.org/zap"
 )
 
+type SamplerType int
+
 const (
+	SamplerTypeUnrecognized SamplerType = iota
+	SamplerTypeProbabilistic
+	SamplerTypeLowerBound
+	SamplerTypeRateLimiting
```

Comment:
```suggestion
	SamplerTypeRateLimiting

```

Reply:
keep blank line between enums and other constants, this is not a logical continuation